Server/Controller/OpenIdConnectController.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using AuthServer.Shared;
+
+namespace AuthServer.Server.Controllers
+{
+    [ApiController]
+    [Route("/auth/oidc")]
+    public class OpenIdConnectController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
+        {
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
+
+        private readonly ILogger<OpenIdConnectController> _logger;
```

Comment:
Codacy found an issue: [Remove this unread private field '_logger' or refactor the code to use its value.](https://app.codacy.com/gh/LukasReschke/AuthServer/pullRequest?prid=6395991)